uportal-war/src/main/java/org/jasig/portal/groups/pags/testers/ThemeNameEqualsIgnoreCaseTester.java

```diff
@@ -48,6 +51,21 @@
     private final IUserLayoutStore userLayoutStore;
     private final IStylesheetDescriptorDao stylesheetDescriptorDao;
 
+    public ThemeNameEqualsIgnoreCaseTester(IPersonAttributesGroupTestDefinition definition) {
+        final String themeTestValue = definition.getTestValue();
+        assert StringUtils.isNotBlank(themeTestValue);
+        this.themeTestValue = themeTestValue;
+        this.applicationContext = ApplicationContextLocator.getApplicationContext();
+        this.portalRequestUtils = applicationContext.getBean(IPortalRequestUtils.class);
+        this.profileMapper = applicationContext.getBean("profileMapper", IProfileMapper.class);
+        this.userLayoutStore = applicationContext.getBean("userLayoutStore", IUserLayoutStore.class);
+        this.stylesheetDescriptorDao = applicationContext.getBean("stylesheetDescriptorDao", IStylesheetDescriptorDao.class);
+    }
```

Comment:
I updated the other 2 places where you made this observation, but I'm going to pass on this one.

* The other 2 places were doing some actual work;  this one is just assigning values to member variables, which is pretty natural in a constructor.
* Those fields are marked final -- and not even by me -- so i'd have to "un-final" them (which I'd prefer not to do) in order to set them in an init() method or somesuch.